src/store/options.js

```diff
@@ -156,6 +145,14 @@ const Options = {
   },
 };
 
+export const SYNC_OPTIONS = Object.keys(Options).filter(
+  (key) => !LOCAL_OPTIONS.includes(key),
+);
+
+export const REPORT_OPTIONS = SYNC_OPTIONS.filter(
+  (key) => !PROTECTED_OPTIONS.includes(key),
+).concat(['filtersUpdatedAt']);
+
```

Comment:
This is little hard to follow. Maybe we could use `Set.prototype.difference` to deal with that?

Reply:
It is not a difference. It is just filtering out protected options and adding `filersUpdatedAt`, but I can use the spread operator; it might be more readable. This issue is related to formatting.